src/GudhUI/model/Model.h

```diff
@@ -307,9 +307,15 @@ class Model {
  private:
   void run_chomp() {
     save_complex_in_file_for_chomp();
+    std::cout << "POUET" << std::endl;
     std::cout << "Call CHOMP library\n";
-    int returnValue = system("utils/homsimpl chomp.sim");
-    std::cout << "CHOMP returns" << returnValue << std::endl;
+    int returnValue = system("which homsimpl");
+    if (returnValue != 0) {
+        std::cout << "CHOMP must be installed and available in PATH for this functionnality." << std::endl;
+        return;
+    }
+    returnValue = system("homsimpl chomp.sim");
+    std::cout << "CHOMP returns " << returnValue << std::endl;
```

Comment:
I think it might be enough to test the returnValue of `system("homsimpl chomp.sim")`, and if it is non-zero, explain that homsimpl (from CHOMP) failed, which could happen in particular if it isn't installed or in the PATH. I am a bit wary of using `which`, but maybe it has advantages?

Reply:
https://github.com/VincentRouvreau/gudhi-devel/commit/4d8dc4477942d196a304e71db4918ac71862e597